test/deploy.sh

```diff
@@ -0,0 +1,23 @@
+#/bin/bash
+set -e
+ORG=${ORG:-hsldevcom}
+yarn install
+
+docker run -d -e CONFIG=hsl -p 127.0.0.1:8080:8080 $ORG/digitransit-ui:ci-$TRAVIS_COMMIT
+wget -N http://chromedriver.storage.googleapis.com/2.29/chromedriver_linux64.zip
```

Comment:
Oh, we are actually running tests here, could those be run in parallel with the visual tests in one stage?